x/launch/keeper/msg_server_edit_chain_test.go

```diff
@@ -129,6 +188,47 @@ func TestMsgEditChain(t *testing.T) {
 				false,
 				false,
 				false,
+				0,
+				false,
+			),
+			err: profiletypes.ErrCoordInvalid,
+		},
+		{
+			name: "chain already has campaign",
+			msg: sample.MsgEditChain(coordAddress, launchIDHasCampaign,
+				false,
+				false,
+				false,
+				false,
+				true,
+				0,
+				false,
+			),
+			err: types.ErrChainHasCampaign,
+		},
+		{
+			name: "campaign does not exist",
+			msg: sample.MsgEditChain(coordAddress, launchID2,
+				false,
+				false,
+				false,
+				false,
+				true,
+				999,
+				false,
+			),
+			err: campaigntypes.ErrCampaignNotFound,
+		},
+		{
+			name: "campaign does not exist",
+			msg: sample.MsgEditChain(coordAddress, launchID2,
+				false,
+				false,
+				false,
+				false,
+				true,
+				invalidCampaignID,
```

Comment:
```suggestion
				campaignDifferentCoordinator,
```
Would make more explicit